### ISSUE TITLE
Weekly scrum - Nov 8th, 2024

### ISSUE THREAD

**Issue by Samimnif** [jalalmourad/SYSC4806_Group20#2]
- what have I done this week? (with link(s) to the relevant GitHub Issue(s))
- what will I do next week? (with link(s) to the relevant GitHub Issue(s))
- what is holding me back? (if applicable)
Expected length: 1-2 sentences per item

**Comment by Samimnif** [comment]
* what have I done this week? (with link(s) to the relevant GitHub Issue(s))
  * Created the Professor Model #3 
  * Added REST Endpoints to be able to create a new Professor user through POST Request #5  
  * Added Professor Profile HTML Template that shows the Professor Profile and the List of Topics created by him #6 
* what will I do next week? (with link(s) to the relevant GitHub Issue(s))
  * I will be working on more functionalities, such as deleting professor, deleting topic, editing topic, and improving the template. layouts.
  * We will meet with the team again and coordinate the features that must be added.
* what is holding me back? (if applicable)
  * NA
 <hr>

**Comment by Ibtasam-Rasool** [comment]
what have I done this week?  
- worked on topic page, one is now for viewing and one for students with student amount and added in topic #9, also worked on integrating changes. 
what will I do next week? 
- will work on connecting pages together right now you need to manually enter URLs for some functionality. Improving services and pages.
what is holding me back? N/A

**Comment by jalalmourad** [comment]
1. what have I done this week? (with link(s) to the relevant GitHub Issue(s))
Created the Student, Prof, Topic Model https://github.com/jalalmourad/SYSC4806_Group20/issues/1#issue-2633537775
Created the Azure Pipeline: https://github.com/users/jalalmourad/projects/1/views/1?pane=issue&itemId=85918055
Created CRUD repo's:
https://github.com/users/jalalmourad/projects/1/views/1?pane=issue&itemId=85918390
Created Topic View:
https://github.com/users/jalalmourad/projects/1/views/1?pane=issue&itemId=86571902
Created Remove and View Topic:
https://github.com/users/jalalmourad/projects/1/views/1?pane=issue&itemId=86571780
2. what will I do next week? (with link(s) to the relevant GitHub Issue(s))
I will be working on adding more features, and performing some debugging. 
3. what is holding me back? (if applicable)
NA

**Comment by hadisrour6** [comment]
what have I done this week? (with link(s) to the relevant GitHub Issue(s))
- Created index.html
- Created index.js
- Created project model
what will I do next week? (with link(s) to the relevant GitHub Issue(s))-
- Creating test cases for our models, pipelines and CRUD ensuring everything is working properly
-  Work on adding more features to the application specifically the log in system 
what is holding me back? (if applicable)
-  N/A

**Comment by aarongabor1** [comment]
This week I created the main Topic model that is used to store projects in the database. I also created the ProjectRestriction and Status Enums. I created addProject.html and addProject.js to allow the user to fill out an HTML form to create a new project topic and have that information collected and than sent to the respective controller. 

Next week I will work on creating new features for the project and assisting in debugging any issues that may arise.
